scripts/presubmit.sh

```diff
@@ -38,11 +41,12 @@ if [ $? -ne 0 ]; then
    echo "✋ to enable import cleanup. Import cleanup skipped."
 else
    echo "🧽 Format with goimports"
-   goimports -w $(echo $SOURCE_DIRS)
+   # Find all non-generated Go files. This works by excluding
+   # files that start with the official "generated file" header.
+   # See https://github.com/golang/go/issues/13560#issuecomment-288457920
+   grep -L -HR "^\/\/ Code generated .* DO NOT EDIT\.$" --include="*.go" "${ROOT}" | xargs goimports -w
```

Comment:
returns no results on my machine

```
✦ ❯ grep -L -HR "^\/\/ Code generated .* DO NOT EDIT\.$" --include="*.go" "${ROOT}"                     

```

Reply:
Alright, I give up. That doesn't make any sense. Is `${ROOT}` set when running this? Did you try running `presubmit.sh`? What version of `grep` is installed on Mac? Sorry for the inconvenience :disappointed:, again, I really did think it would work this time.

Reply:
ok, verified this is working for me now.